test/wsdl/default_namespace.wsdl

```diff
@@ -43,15 +37,6 @@
         <soap:body use="literal" encodingStyle="http://schemas.xmlsoap.org/soap/encoding/"/>
       </wsdl:output>
     </wsdl:operation>
-    <wsdl:operation name="prefixed-MyOperation">
```

Comment:
why is this being removed?

Reply:
In an earlier change I added unittests using a changed version of the existing `default_namespace.wsdl` with added operations that contained non identifier characters. I decided to move these to a new wsdl (`non_identifier_chars_in_operation.wsdl`) as to not possibly changing the behaviour of other tests (altough they didn't fail).

Reply:
ideally we shouldn't be changing existing tests.  Period.  Can you revert this?

Reply:
@jsdevel: I think we agreed already: this change reverts my earlier commit https://github.com/vpulim/node-soap/pull/978/commits/e327a60cd70930c5e0fa258d5c2f737aa9a3fdbd

Reply:
ahh, I see.  Thanks for clarifying @djjanse !